docs/appendices/release-notes/unreleased.rst

```diff
@@ -114,40 +114,6 @@ Changes
 Fixes
 =====
 
-- Improved the internal dynamic batching mechanism for operations like ``INSERT
-  INTO FROM QUERY`` and ``COPY FROM``. It should more aggressively throttle
-  these operations in case of memory pressure, reducing the chance of them
-  failing with a ``CircuitBreakingException``.
-
-- Reduced the default :ref:`initial concurrency limit
-  <overload_protection.dml.initial_concurrency>` for operations like ``INSERT
-  INTO FROM QUERY`` from 50 to 5. This is closer to the behavior before 4.6.0.
-  If the nodes have spare capacity for a higher concurrency the effective
-  concurrency limit will grow dynamically over time, but it will start out
-  lower to avoid overloading a cluster with an initial spike of internal
-  requests.
-
-- Added various :ref:`overload protection <overload_protection>` settings to
-  control the concurrency of operations like ``INSERT INTO FROM QUERY``.
-
-- Fixed an issue that caused ``ALTER TABLE <tbl> ADD COLUMN <columName> INDEX
-  USING FULLTEXT`` statements to ignore the ``INDEX USING FULLTEXT`` part.
-
-- Fixed a performance regression introduced in 4.2 which could cause queries
-  including joins, virtual tables and ``LIMIT`` operators to run slower than
-  before.
-
-- Fixed an issue that caused ``INSERT INTO`` statements to fail on partitioned
-  tables where the partitioned column is generated and the column and value are
-  provided in the statement.
-
- - Fixed an issue that caused ``NullPointerException`` when inserting into
-   previously altered tables that were partitioned and had generated columns.
-
 - Fixed an issue that caused ``UNION ALL`` statements to succeed or throw
```

Comment:
This was not backported, is this intentionally ?

Reply:
@jeeminso ^^ Fix only related to master?

Reply:
@mkleen, @seut sorry I think this is supposed to be backported. I was not aware of the process to tag bug fixes

Reply:
ok, no problem. @Seut how should we handle that ?

Reply:
@jeeminso @mkleen ok so lets backport it then ;)